src/main/scala/se/nimsa/dicom/streams/ModifyFlow.scala

```diff
@@ -81,15 +83,18 @@ object ModifyFlow {
     * Modifications can either be supplied as input arguments or passed at any time on the stream encapsulated in a
     * [[se.nimsa.dicom.streams.ModifyFlow.TagModificationsPart]]
     *
-    * @param modifications Any number of [[se.nimsa.dicom.streams.ModifyFlow.TagModification]]s each specifying a tag path, a modification function, and
-    *                      a Boolean indicating whether absent values will be inserted or skipped.
+    * @param modifications a sequence of tag modifications each with its on trigger function on current tag path, and
+    *                      modification function for element value when triggered
+    * @param insertions    a sequence of tag insertion specifications each with a tag path pointer to the element to
+    *                      be replaced or inserted, and the new value
     * @return the modified flow of DICOM parts
     */
-  def modifyFlow(modifications: TagModification*): Flow[DicomPart, DicomPart, NotUsed] =
+  def modifyFlow(modifications: Seq[TagModification] = Seq.empty, insertions: Seq[TagInsertion] = Seq.empty): Flow[DicomPart, DicomPart, NotUsed] =
```

Comment:
Modify flow now separates between modifications (supports tag trees) and insertions (only tag paths allowed)